app/schemas/models/oauth2identity.schema.js

```diff
@@ -0,0 +1,31 @@
+const schema = require('./../schemas')
+
+const OAuth2IdentitySchema = schema.object({
+  description: 'An OAuth2 Identity',
+  title: 'OAuth2 Identity',
+  required: ['provider']
+})
+_.extend(OAuth2IdentitySchema.properties, {
+  provider: { type: 'string' },
```

Comment:
can we make this enum so we know what all providers we support?

Reply:
It can be the slug of any provider from OAuthProviders